access/templates/access/graded_form.html

```diff
@@ -28,6 +31,7 @@
 
 			{% if not field.field.group_errors %}
 			{% if result.accepted %}
+			<!-- Points badge -->
```

Comment:
These comments could be template comments, and thus wouldn't be part of the data visible to the user. (e.g. `{# foobar #}`), except if the goal is to be able to see these in the html source.